stats.js

```diff
@@ -66,18 +67,7 @@ const getTestingData = async () => {
   } catch (err) {
     maxAge = defaultMaxAge
```

Comment:
Unrelated to the rolling averages, but I see that errors here are swallowed without logging anything. 

If this was failing for some reason it'd quietly use the default and carry on with no evidence anything was wrong - which fits what we've seen happening on the actual APIs. 

I don't know anything about how this is deployed etc and maybe you expect it to error in envs where the parameter has deliberately not been set (e.g. `prd` currently), but maybe errors here should be logged if this is deployed in `dev` environment?